packages/drawer/Drawer.tsx

```diff
@@ -22,28 +24,41 @@ export class Drawer extends Component<DrawerProps> {
   static get props() {
     return {
       position: prop.string( {
-        attribute: true
+        attribute: {
+          source: true
+        }
       } ),
       visible: prop.boolean( {
-        attribute: true
+        attribute: {
+          source: true
+        }
+      } ),
+      floating: prop.boolean( {
+        attribute: {
+          source: true
+        },
+        initial: true
       } )
     };
   }
 
   position = 'top';
```

Comment:
I really don't have time to reference everything to everyone, you can follow my advice as person who knows the internals of the library or feel free to google it 4 yourself ( good start is  project github issues and PR's ). Once again It is going away in next iteration of skate + it is bad design pattern to use ( when setting initial prop values we use `default`, we already talked about that within team as you may remember )